mozci/data/sources/taskcluster/__init__.py

```diff
@@ -147,7 +148,17 @@ def run_push_existing_classification(self, branch, rev, environment):
             )
 
         try:
-            return response["push"]["classification"]
+            r = requests.get(url)
```

Comment:
I would either add `allow_redirects=Trues` or add a comment explaining we don't use the internal method because it may fail on redirections

Reply:
Ah, that's the name, I tried using `follow_redirects` but the parameter didn't exist so I simply lazily abandoned the idea of adding it :P

Reply:
Done both with ee83b1a and 765b355.